### PR TITLE
Make interview workflow complete and test in end to end

### PR DESCRIPTION
**Pull request by harshitsharma7017**
Api's to seperate completed and current interviews for interviewee

### CODE REVIEW

File: src/constants/interview.constant.ts [hunk 1]
```diff
@@ -1,4 +1,9 @@
 export const INTERVIEW = "/interview";
-export const CREATE_INTERVIEW_END_POINT = "/:interviewee";
+export const CREATE_INTERVIEW_END_POINT = "/:_id";
 export const UPDATE_INTERVIEW_END_POINT = "/:interview_id";
```
Comment (Akhil-chaudhary) [inline src/constants/interview.constant.ts:3]: make this interviewee_id

Comment (Akhil-chaudhary) [reply src/constants/interview.constant.ts:3]: keep these things consistent

Comment (harshitsharma7017) [reply src/constants/interview.constant.ts:3]: Ok sir

Comment (harshitsharma7017) [reply src/constants/interview.constant.ts:3]: sir ye tushar sir se mna kra tha, kyuki purani api me conflict aara tha

Comment (Akhil-chaudhary) [reply src/constants/interview.constant.ts:3]: shi karna toh padega

Comment (harshitsharma7017) [reply src/constants/interview.constant.ts:3]: done sir, please review

**Pull request merged**